### PR TITLE
Replacing transcript with coding sequence in plant metabolic modeling

### PR DESCRIPTION
**Pull request by samseaver**
Again, let me check with plant team before merging

### CODE REVIEW

File: apps/build_plant_metabolic_model/display.yaml [hunk 1]
```diff
@@ -3,9 +3,9 @@
 #
 name     : Build Plant Metabolic Model
 subtitle : |
-    Build a metabolic model for a plant based on annotated sequences of transcripts.
+    Build a metabolic model for a plant based on annotated sequences of coding sequences.
```
Comment (nlharris) [inline apps/build_plant_metabolic_model/display.yaml]: "annotated sequences of coding sequences" seems redundant. Maybe just "annotated coding sequences"?

Comment (samseaver) [inline apps/build_plant_metabolic_model/display.yaml]: "annotation of coding sequences" ?

Comment (nlharris) [inline apps/build_plant_metabolic_model/display.yaml]: Well, that sounds like it means that all that is annotated is where the coding sequences are. Is that the case?

Comment (samseaver) [inline apps/build_plant_metabolic_model/display.yaml]: Can you rephrase?

Remember this is an app, so it covers both annotation and metabolic modeling. I thought that the passive voice implied that annotation would already be done when running the app.

Comment (nlharris) [inline apps/build_plant_metabolic_model/display.yaml]: What was wrong with "annotated coding sequences"?

Comment (samseaver) [inline apps/build_plant_metabolic_model/display.yaml]: Never mind, I updated it.
